File: Api_Azure_JavaSpringBoot_MonsterAsp_MySql/Api/dto/BtsDto.java
package cz.sliva.nobodywebapi.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class BtsDto {

    @JsonProperty("id")
    private int id;

    @JsonProperty("cellid")
    private int cellid;

    @JsonProperty("physcid")
    private int physcid;

    @JsonProperty("tac")
    private int tac;

    @JsonProperty("band")
    private int band;

    @JsonProperty("gsmcid")
    private int gsmcid;

    @JsonProperty("datum")
    private LocalDate datum;

    @JsonProperty("okres")
    private String okres;

    @JsonProperty("adresa")
    private String adresa;

    @JsonProperty("adrId")
    private int adrId;
}

File: Api_Azure_C#_ASP_MSSQL/Api/Dto/BtsDto.cs
namespace NoBodyApi2 {
    public class BtsDto {
        public int Id {
            get; set;
        }
        public int? Cellid { get; set; }
        
        public int? Physcid { get; set; }

        public int? Tac {
            get; set;
        }
        public int? Band {
            get; set;
        }
        public int? Gsmcid {
            get; set;
        }
        public DateOnly? Datum {
            get; set;
        }

        public string Okres {
            get; set;
        }
        public string Adr {
            get; set;
        }

    }
}


